src/components/Patient/PatientDetailsTab/PatientUsers.tsx

```diff
@@ -287,7 +311,7 @@ export const PatientUsers = (props: PatientProps) => {
                     <AlertDialogAction
                       data-cy="patient-user-remove-confirm-button"
                       onClick={() => removeUser(user.id)}
-                      className="bg-destructive text-destructive-foreground hover:bg-destructive/90"
+                      className="bg-red-500 text-white hover:bg-red-600"
```

Comment:
Let's not make this change for now. We shouldn't be overriding styles like this anyways, instead relying on variants.

Reply:
@rajku-dev You can use

```suggestion
                      className="cn(buttonVariants({ variant: "destructive" }))"
```
instead

Reply:
made the changes as requested